capstone/src/main/java/com/google/sps/servlets/BlackOwnedRestaurantsDataServlet.java

```diff
@@ -51,4 +56,31 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String json = gson.toJson(restaurantNames);
     response.getWriter().println(json);
   }
-}
\ No newline at end of file
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String restaurantName = request.getParameter("businessName");
+    boolean blackOwned = true;
+    boolean smallBusiness = false;
+    // stores tags like "pizza" or "chinese"
+    // implemented as hashset for faster lookup
```

Comment:
I think this comment is probably unnecessary. Usually you'll talk about a data structure if it's weird or unintuitive, but I think a hashset is pretty straightforward.

Reply:
makes sense